federation/pkg/dnsprovider/providers/aws/route53/rrset.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2016 The Kubernetes Authors All rights reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package route53
+
+import (
+	"github.com/aws/aws-sdk-go/service/route53"	
+	"k8s.io/kubernetes/federation/pkg/dnsprovider/rrstype"
+)
+
+type ResourceRecordSet struct {
+	impl   *route53.ResourceRecordSet
+	rrsets *ResourceRecordSets
+}
+
+func (rrset ResourceRecordSet) Name() string {
+	return *rrset.impl.Name
+}
+
+func (rrset ResourceRecordSet) Rrdatas() []string {
```

Comment:
I would absolutely agree, only rrdatas is a pretty well-known DNS term, used by the DNS RFC, Google Cloud DNS, AWS Route 53, bind, and most other DNS things.  So I suggest that we leave it the way it is.